static/js/components/ProgramSelector.js

```diff
@@ -0,0 +1,119 @@
+// @flow
+import React from 'react';
+import _ from 'lodash';
+import { ReactPageClick } from 'react-page-click';
+
+import type { DashboardState } from '../flow/dashboardTypes';
+import type {
+  ProgramEnrollment,
+  ProgramEnrollmentsState,
+} from '../flow/enrollmentTypes';
+
+export default class ProgramSelector extends React.Component {
+  props: {
+    currentProgramEnrollment:    ProgramEnrollment,
+    enrollments:                 ProgramEnrollmentsState,
+    enrollDialogVisibility:      boolean,
+    enrollSelectedProgram:       ?number,
+    dashboard:                   DashboardState,
+    programSelectorOpen:         boolean,
+    setCurrentProgramEnrollment: (enrollment: ProgramEnrollment) => void,
+    setEnrollDialogVisibility:   (open: boolean) => void,
+    setEnrollSelectedProgram:    (programId: number) => void,
+    setProgramSelectorOpen:      (open: boolean) => void,
+  };
+
+  handleProgramChange = (event: any): void => {
+    const { enrollments, setCurrentProgramEnrollment } = this.props;
+    const value = parseInt(event.target.value);
+
+    const enrollment = enrollments.programEnrollments.find(enrollment => enrollment.id === value);
+    setCurrentProgramEnrollment(enrollment);
+  };
```

Comment:
No but it will be used in the next PR for choosing between programs to enroll in